tests/locale/keymap_or_locale.pm

```diff
@@ -0,0 +1,93 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Keyboard layout test in console and gdm after boot
+# Maintainer: Martin Loviska <mloviska@suse.com>
+
+use base "opensusebasetest";
+use strict;
+use testapi;
+use utils;
+
+sub verify_default_keymap_textmode {
+    my ($test_string, $tag, $tty, $console) = @_;
+
+    send_key($tty)           if defined($tty);
+    select_console($console) if defined($console);
+    type_string($test_string);
+    assert_screen($tag, 40);
```

Comment:
I guess we can have default timeout here, to avoid magic numbers. WDYT?

Reply:
agreed, magic number has been removed.